config/locales/doc_auth/en.yml

```diff
@@ -86,8 +86,7 @@ en:
         network_error: We are having technical difficulties on our end. Please try to
           submit your images again later.
         no_liveness: Try taking new pictures.
-        selfie_failure: Try taking your photos again. Make sure all of your photos are
-          clear and in focus.
+        selfie_failure: We couldn’t verify the photo of yourself. Try taking a new picture.
```

Comment:
@kellular (or anyone) I'm not seeing translations? I'll keep looking.

Reply:
@charleyf They might have been accidentally removed the last time we worked on this in-line error, I can't recall. But here are the DOS translations:

Spanish: No pudimos verificar su foto. Intente tomar una nueva foto.
French: Nous n’avons pas pu vérifier votre photo. Essayez de prendre une nouvelle photo.

Reply:
Thanks! Added those, I don't see anywhere to put the simplified Chinese yet. I would expect a file like `doc_auth/zh-CN.yml`.

Reply:
@charleyf , it's zh.yml.

Reply:
@kellular yes, please include Chinese translations going forward

Reply:
Added the chinese translation

Reply:
@charleyf thanks for adding the chinese translation - could you please add a screenshot to the PR so I can submit this for DOS LQA approval?

Reply:
Uploaded a screenshot to the PR description.